python/ray/train/_internal/state/schema.py

```diff
@@ -5,13 +5,17 @@
 from ray.dashboard.modules.job.pydantic_models import JobDetails
 from ray.util.annotations import DeveloperAPI
 
+MAX_ERROR_STACK_TRACE_LENGTH = 50000
+
 
 @DeveloperAPI
 class RunStatusEnum(str, Enum):
     """Enumeration for the status of a train run."""
 
-    # The train run has started
+    # (Deprecated) The train run has started
     STARTED = "STARTED"
```

Comment:
nit: For deprecated things it's recommended to document the replacement (RUNNING in this case)